pilot/pkg/security/authz/builder/builder_test.go

```diff
@@ -77,74 +66,6 @@ func simpleGlobalPermissiveMode() *model.Config {
 	return cfg
 }
 
-func TestIsRbacEnabled(t *testing.T) {
```

Comment:
Do we have a test covering what is delete here?

Reply:
The test is just moved to authorization_test.go.